problems/objects.js

```diff
@@ -151,15 +187,32 @@ const authorScores = () => {};
  * @returns {string} The full name of person with best score.
  */
 
-const bestScore = () => {};
+const bestScore = (submissions) => {
+  let highestScore = -Infinity;
+  let name = "";
+  for (let i = 0; i < submissions.length; i++) {
+    if (submissions[i].score > highestScore) {
+      highestScore = submissions[i].score;
+      name = submissions[i].firstName + " " + submissions[i].lastName;
```

Comment:
I'd recommend saving the object as the variable, and then formatting the name response at the end.